Gems/Camera/Code/Source/ViewportCameraSelectorWindow_Internals.h

```diff
@@ -58,6 +58,8 @@ namespace Camera
         private:
             AZStd::vector<CameraListItem> m_cameraItems;
             AZ::EntityId m_sequenceCameraEntityId;
+            AZ::EntityId m_lastActiveCamera;
+            bool m_firstEntry = true;
```

Comment:
question: I'm just curious why we need `m_firstEntry` - a comment here and where it's used could be useful